docs/modules/ROOT/pages/how-to-guides/managing-applications/proc-managing_applications.adoc

```diff
@@ -0,0 +1,91 @@
+= Managing applications in secure development environment
+
+This feature enhances the handling of the sensitive content within the development environment. You can efficiently address security fixes while maintaining confidentiality and facilitating the recreation of applications.
+
+Key capabilities of this feature include:
+
+* **Clone affected application:** Cloning affected applications into a secure workspace, enabling developers to tackle security issues without exposing sensitive information.
+
+* **Recreate application with limited visibility:** Recreating applications while minimizing the awareness of their existence, ensuring a secure environment for working on security fixes.
+
+* **Release secure fixes on components:** Release secure fixes on specific components within existing applications, simplifying the process of addressing vulnerabilities without affecting the entire application.
```

Comment:
The bullets sound pretty similar to me. There might be opportunity to simplify here to one or two bullets, or diversify them a bit. If this procedure doesn't talk about releasing, should that be called out in the third bullet?

I think you can use recreate as you have, but typically I see it with a hyphen to keep it distinct from being confused with a form of the different word, recreation. Not sure what the stylistic rules/guidelines are.